packages/kbn-expandable-flyout/src/index.test.tsx

```diff
@@ -45,12 +45,16 @@ describe('ExpandableFlyout', () => {
 
   it('should render right section', () => {
```

Comment:
Okay i understand. Initially, I was taken aback with the design of having one provider per flyout, since I assumed from the redux state that there will be miltiple flyout handled by one provider. But in your other comment you have stated that end goal is to have one provider for all flyouts, I think then these tests will be required. For this PR, this comment can be ignored.

Thanks

Reply:
Now I'm even more confused! There is currently a single instance of a flyout per expandable flyout provider and this is not going to change. If you're talking about [this comment](https://github.com/elastic/kibana/pull/176457#discussion_r1484473020), I said _the goal it NOT to have one provider only_.
I feel like we might be talking about a different provider level (expandable flyout provider vs internal context provider vs redux provider). To be super clear, this is what we should have:
```
<ExpandableFlyoutProvider>
    <SingleFlyout />
</ ExpandableFlyoutProvider>
```

I see that I could add test to check that memory mode doesn't store anything in the url and that url mode indeed stores things in the url.
Though testing multiple providers for multiple flyouts seem more some UI integration test than unit tests no? Can I watch the changes to the url if I'm only rendering a single component in Jest?